### DISCUSSION TITLE
Dbeaver does not check os version

### DISCUSSION THREAD

**Discussion by m3dbedb** [dbeaver/dbeaver#11369] [Q&A] [answered]
#### System information: 
- Operating system windows 7 32
- DBeaver version 7.3.5

#### Describe the problem you're observing:
i download universal binary, but it does not ask windows version and installs 64 bit

#### Steps to reproduce, if exist: double click on dbeaver.exe

**Comment by uslss** [comment, marked as answer]
We stopped delivering 32-bit packages of DBeaver CE since 6.0.1. Mostly because Eclipse platform stopped supporting 32-bit and because Java stopped this was well.
